### PR TITLE
pages*: use manned.org as much as possible (part 1)

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by debghs — APPROVED** [review]

File: pages/linux/pacman.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > Arch Linux package manager utility.
 > See also: `pacman-database`, `pacman-deptest`, `pacman-files`, `pacman-key`, `pacman-mirrors`, `pacman-query`, `pacman-remove`, `pacman-sync`, `pacman-upgrade`.
 > For equivalent commands in other package managers, see <https://wiki.archlinux.org/title/Pacman/Rosetta>.
-> More information: <https://man.archlinux.org/man/pacman.8>.
+> More information: <https://manned.org/pacman.8>.
```
Comment (acuteenvy) [inline pages/linux/pacman.md:6]: According to the current style guide, linking to the author's provided man page is preferred.
> On the `More information` link line, we prefer linking to the author's provided documentation of the command-line reference ***or the man page***.

**Review by kant — APPROVED** [review]
spanish chapter: LGTM

**Comment by Managor** [comment]
I just saw a page you might have missed: `adduser` https://github.com/tldr-pages/tldr/blob/main/pages/linux/adduser.md

**Comment by Managor** [comment]
I did `find . -type f -exec awk '!/manned/ && /man/ && /More information/ ' {} \; > ~/log` on all the pages, sorted them and filtered by unique and there were a couple that caught my eye
- manpages.ubuntu.com
- manpages.debian.org
- manpages.opensuse.org
- manpagez.com
- man.freebsd.org
- man.openbsd.org
- man.netbsd.org
- man.voidlinux.org
- man.cx

Full list of the resulting lines if you want to check it 
[log3.txt](https://github.com/user-attachments/files/17024542/log3.txt)

**Comment by Managor** [comment]
Couple more 
- www.freebsd.org
- www.freedesktop.org
- www.mkssoftware.com
- www.x.org
- x.org
Although I'm not sure about some of these. Some of these are kinda first party sources but they contain manpages that can also be found in manned

Then there are www.unix.com links which return 404

**Comment by Managor** [comment]
I ran all more information links through `lychee`, pruned "Too Many Requests" and "Forbidden" responses, sorted and uniqued.
This is the leftover links that returned 404 or something similar 
[log.txt](https://github.com/user-attachments/files/17035579/log.txt)

**Comment by Managor** [comment]
Yeah, it wasn't intended for this PR. I just thought it would be nice if all these manpage TODOs were collected in the same place.

**Review by reinhart1010 — APPROVED** [review]

**Review by debghs — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**